Index: local.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>## This file must *NOT* be checked into Version Control Systems,\r\n# as it contains information specific to your local configuration.\r\n#\r\n# Location of the SDK. This is only used by Gradle.\r\n# For customization when using a Version Control System, please read the\r\n# header note.\r\n#Mon Oct 09 21:27:31 EDT 2023\r\nsdk.dir=/Users/zacharysikka/Library/Android/sdk\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/local.properties b/local.properties
--- a/local.properties	(revision 6f657d9d2004c3ffbe9735c257b6e4ffa046cf89)
+++ b/local.properties	(date 1699559622929)
@@ -4,5 +4,5 @@
 # Location of the SDK. This is only used by Gradle.
 # For customization when using a Version Control System, please read the
 # header note.
-#Mon Oct 09 21:27:31 EDT 2023
-sdk.dir=/Users/zacharysikka/Library/Android/sdk
+#Thu Nov 09 14:53:42 EST 2023
+sdk.dir=C\:\\Users\\linde\\AppData\\Local\\Android\\Sdk
Index: .gradle/buildOutputCleanup/cache.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>#Mon Oct 09 14:47:30 EDT 2023\r\ngradle.version=8.0\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/.gradle/buildOutputCleanup/cache.properties b/.gradle/buildOutputCleanup/cache.properties
--- a/.gradle/buildOutputCleanup/cache.properties	(revision 6f657d9d2004c3ffbe9735c257b6e4ffa046cf89)
+++ b/.gradle/buildOutputCleanup/cache.properties	(date 1699559629911)
@@ -1,2 +1,2 @@
-#Mon Oct 09 14:47:30 EDT 2023
+#Thu Nov 09 14:53:49 EST 2023
 gradle.version=8.0
